### PR TITLE
NOISSUE - Fix provision tool connect error handling

### PR DESCRIPTION
**Pull request by dborovcanin**
Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

### What does this do?
This pull request adds error handling for connect errors.
Remove unused params from MQTT bench client.

### List any changes that modify/break current functionality
There are no such changes.

### Have you included tests for your changes?
No.

### Did you document any new/modified functionality?
No.

### CODE REVIEW

**Review by manuio — APPROVED** [review]
LGTM

File: tools/provision/provision.go [hunk 1]
```diff
@@ -215,13 +215,18 @@ func Provision(conf Config) {
 	fmt.Printf("# List of channels that things can publish to\n" +
 		"# each channel is connected to each thing from things list\n")
 	for i := 0; i < conf.Num; i++ {
-		for j := 0; j < conf.Num; j++ {
+		// Creating a new routine for each connect
+		// might be heavy on the network.
```
Comment (drasko) [inline tools/provision/provision.go:219]: I would avoid fullstop on the comments

**Review by drasko — CHANGES_REQUESTED** [review]

File: tools/provision/provision.go (hunk 1 above)
Comment (dborovcanin) [reply tools/provision/provision.go:219]: I'm following natural language (which is, roughly speaking in this context, the definition of comments) rules. If this wasn't a full sentence, I'd leave it with no punctuation. If you insist, I'll change it, but please have in mind that this is in sync with most of the Mainflux code comments (though other comments embraced this style mostly due to GoDoc and native Go code). I prefer it this way, as well.

Comment (drasko) [reply tools/provision/provision.go:219]: OK, I leave it to you. Accepted.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**